Modules/System/Email/src/Message/EmailMessage.Codeunit.al

```diff
@@ -115,6 +151,26 @@ codeunit 8904 "Email Message"
         Recipients := EmailMessageImpl.GetRecipients(RecipientType);
     end;
 
+    /// <summary>
+    /// Sets the recipents of a certain type of the email message.
+    /// </summary>
+    /// <param name="RecipientType">Specifies the type of the recipients.</param>
+    /// <param name="Recipients">Specifies the list of the recipients' email addresses as a semicolon (;) separated list.</param>
+    procedure SetRecipients(RecipientType: Enum "Email Recipient Type"; Recipients: Text)
+    begin
+        EmailMessageImpl.SetRecipients(RecipientType, Recipients);
+    end;
+
+    /// <summary>
+    /// Sets the recipents of a certain type of the email message.
+    /// </summary>
+    /// <param name="RecipientType">Specifies the type of the recipients.</param>
+    /// <param name="Recipients">Specifies the list of the recipients' email addresses.</param>
+    procedure SetRecipients(RecipientType: Enum "Email Recipient Type"; Recipients: list of [Text])
```

Comment:
Just food for thought: isn't `AddRecipient(RecipientType: Enum "Email Recipient Type"; Recipient: Text)` a bit more useful? Or at least useful as much?

Is there a scenario where the whole list of recipients will be changed?

Reply:
Good idea, I've added it